test/models/report_test.rb

```diff
@@ -3,7 +3,68 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  setup do
+    @report1 = FactoryBot.create(:report)
+    @report2 = FactoryBot.create(:report)
+    @report3 = FactoryBot.create(:report)
+    @report4 = FactoryBot.create(:report)
+  end
+
+  test '#editable? should return true when target user tries to edit a report created by target user' do
+    target_user = FactoryBot.create(:user)
+    report = FactoryBot.create(:report, user: target_user)
+
+    assert report.editable?(target_user)
+  end
+
+  test '#editable? should return false when target user tries to edit a report created by another user' do
+    target_user = FactoryBot.create(:user)
+    another_user = FactoryBot.create(:user, email: 'bob@example.com', name: 'Bob')
```

Comment:
テストコード上、emailやnameがbob/Bobである必要はない（デフォルトの名前でもテストはパスする）ので、これでもいいと思います。 

```suggestion
    target_user = FactoryBot.create(:user)
    another_user = FactoryBot.create(:user)
```